polclassifier/params.py

```diff
@@ -32,6 +31,9 @@
 GAMMA_DEFAULT = "scale"
 C_DEFAULT = 4.2
 
+# Randomized search parameters for model KNN
+N_NEIGHBORS = randint(1, 100)
+LEAF_SIZE = randint(1, 100)
 
```

Comment:
Please also add the best hyperparams that you found so the random search doesn't have to be run every time. (see Hai Linh's example from lines 31-33 from this file and then feed those hyperparameters into you KNN model in line 93 of the `models.py` file, similarly to how she did it in her script).